components/script/dom/htmlimageelement.rs

```diff
@@ -113,7 +116,8 @@ impl Runnable for ImageResponseHandlerRunnable {
         let element = self.element.root();
         // Ignore any image response for a previous request that has been discarded.
         if element.generation.get() == self.generation {
-            element.process_image_response(self.image);
+            let url = self.url.clone();
+            element.process_image_response(self.image, &url);
```

Comment:
yes indeed, and I also had to use `&self.image` to prevent `use of moved value: self`